unit-tests/post-processing/test-rotation-filter.py

```diff
@@ -0,0 +1,133 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#temporary fix to prevent the test from running on Win_SH_Py_DDS_CI
+#test:donotrun:dds
+#test:donotrun:!nightly
```

Comment:
Is this test long? why !nightly?

Did you check to see if this test run?

Reply:
Not very long I can remove it

Reply:
Let's do it.

Reply:
Done